tests/test_indexing.py

```diff
@@ -1057,3 +1057,87 @@ def test_fill_diagonal_error():
     arr = dpnp.ones((1, 2, 3))
     with pytest.raises(ValueError):
         dpnp.fill_diagonal(arr, 5)
+
+
+class TestSelect:
```

Comment:
Please move the old `def test_select():` to this class as well or remove it if it is not necessary any more.